docs/installation/getting-started/getting-and-setting-data.md

```diff
@@ -1,11 +1,15 @@
 ---
-category: advanced
-order: 20
+category: getting-started
+order: 70
 ---
 
-{@snippet installation/saving-data/build-autosave-source}
+# Getting and setting data
```

Comment:
Build import was missing here. It should look like that:
```suggestion
{@snippet installation/getting-and-setting-data/build-autosave-source}

# Getting and setting data
```

Reply:
Fixed in https://github.com/ckeditor/ckeditor5/pull/13170/commits/d015ea071852c953517943f1f1d886baa027296b.